### PR TITLE
Backport (of  #33966) to 106X: Adding necessary electron structures for low pT part of slimmedElectron collection

### PR DESCRIPTION
**Pull request by jainshilpi**
This is just a backport of #33966  which adds the needed structures to all the electrons to enable low pT electron analyses using slimmedElectron collection.
Tested on WFs: 136.898 

@bainbrid @cavalari

### CODE REVIEW

**Comment by mariadalfonso** [comment]
please test

File: PhysicsTools/PatAlgos/python/slimming/slimmedElectrons_cfi.py [hunk 1]
```diff
@@ -26,3 +26,6 @@
 
 from Configuration.Eras.Modifier_bParking_cff import bParking
 bParking.toModify(slimmedElectrons,dropCorrections="0")
+bParking.toModify(slimmedElectrons,dropIsolations="0")
+bParking.toModify(slimmedElectrons,dropShapes="0")
+bParking.toModify(slimmedElectrons,dropClassifications="0")
```
Comment (slava77) [inline PhysicsTools/PatAlgos/python/slimming/slimmedElectrons_cfi.py:31]:
```suggestion
bParking.toModify(slimmedElectrons,dropCorrections="0", dropIsolations="0", dropShapes="0", dropClassifications="0")
```
this is simpler to maintain if we need to extend to another modifier

**Comment by slava77** [comment]
test parameters:
- workflows = 136.898,1304.181

**Comment by perrotta** [comment]
backport of #33966

File: PhysicsTools/PatAlgos/python/slimming/slimmedElectrons_cfi.py (hunk 1 above)
Comment (slava77) [reply PhysicsTools/PatAlgos/python/slimming/slimmedElectrons_cfi.py:31]: @jainshilpi 
are you available to update this PR?

Comment (jainshilpi) [reply PhysicsTools/PatAlgos/python/slimming/slimmedElectrons_cfi.py]: Hi Slava, I missed that comment. Just pushed the modification.

**Comment by slava77** [comment]
@cmsbuild please test

File: PhysicsTools/PatAlgos/python/slimming/slimmedElectrons_cfi.py (hunk 1 above)
Comment (slava77) [reply PhysicsTools/PatAlgos/python/slimming/slimmedElectrons_cfi.py]: > Hi Slava, I missed that comment. Just pushed the modification.

Thank you.

**Comment by slava77** [comment]
+reconstruction

for https://github.com/cms-sw/cmssw/pull/33977	311359e 
- code changes are in line with the PR description and the follow up review
    - the updates are consistent with #33966 under the constraints of the no-change policy
- jenkins tests pass and comparisons with the baseline show no differences
- miniAOD output of 136.898 (bParking wf) here compared to a test in #33589 show a small increase in the size of `slimmedElectrons` collection, as expected

**Comment by qliphy** [comment]
+1

**Pull request merged**